modules/6c-labs-code-academic-r.md

```diff
@@ -115,8 +112,8 @@ user_ids <- c("2244994945", "6253282")
 
 users <-
   get_user_profile(
-    user_ids,
-    bearer_token)
+    x = user_ids
```

Comment:
what does this line do?

Reply:
It's the same as the original code. The only difference is that the [main argument of the function `get_user_profile` is called `x`](https://rdrr.io/cran/academictwitteR/man/get_user_profile.html). My suggestion only makes it explicit.

``` r
require(academictwitteR)
#> Loading required package: academictwitteR
user_ids <- c("2244994945", "6253282")

users1 <-
  get_user_profile(
    x = user_ids
  )
#> Processing from 1 to 2

users2 <-
  get_user_profile(
    user_ids,
    bearer_token = get_bearer())
#> Processing from 1 to 2

testthat::expect_equal(users1$id, users2$id)
```

<sup>Created on 2021-07-01 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>